drivers/backup/portworx/portworx.go

```diff
@@ -162,6 +164,25 @@ func (p *portworx) CreateCluster(req *api.ClusterCreateRequest) (*api.ClusterCre
 	return p.clusterManager.Create(context.Background(), req)
 }
 
+func (p *portworx) CreateNewCluster(name string, cloudCred string, kubeconfigPath string, orgID string) (*api.ClusterCreateResponse, error) {
```

Comment:
does it really need to be exported? Can we simple create `ClusterCreateRequest ` outside then call `CreateCluster`?

Reply:
@thiguetta removed `CreateNewCluster()` API call altogether from the interface.